src/destinations/dune.py

```diff
@@ -20,10 +20,15 @@ class DuneDestination(Destination[DataFrame]):
     table_name : str
         The name of the table where the query results will be stored.
 
+    [optional]
+    request_timeout : int
+        Default: 10
+        The request timeout for the dune client.
+
     """
 
-    def __init__(self, api_key: str, table_name: str):
-        self.client = DuneClient(api_key)
+    def __init__(self, api_key: str, table_name: str, request_timeout: int = 10):
```

Comment:
Since we set the default in config, I think we can make it required here...
See this related issue: #104 
```suggestion
    def __init__(self, api_key: str, table_name: str, request_timeout: int):
```